packages/back-end/src/scim/users/updateUser.ts

```diff
@@ -0,0 +1,59 @@
+import { Response } from "express";
+import { getUserById, updateScimUserData } from "../../services/users";
+import { ScimUserPutOrPostRequest } from "../../../types/scim";
+
+export async function updateUser(
+  req: ScimUserPutOrPostRequest,
+  res: Response
+): Promise<Response> {
+  const user = await getUserById(req.params.id);
+
+  const { name, userName, displayName } = req.body;
+
+  if (!user) {
+    return res.status(404).json({
+      schemas: ["urn:ietf:params:scim:api:messages:2.0:Error"],
+      status: "404",
+      detail: "User not found",
+    });
+  }
+
+  try {
+    // TODO: Explore if we should support updating external ID
+    await updateScimUserData(user.id, {
```

Comment:
Do we want to support this? In the app today, we don't support updating a user's name/email, but rather require them to delete the user and create a new one.

Maybe we have to in order to satisfy SCIM?

Reply:
I think updating email at a minimum is required. Not sure what problems updating name could introduce

Reply:
We can't support this.  An organization API key can only make changes to the organization itself, not the user model.  Users may be part of multiple organizations.